advocacy_docs/supported-open-source/patroni/03-rhel8_quick_start.mdx

```diff
@@ -0,0 +1,277 @@
+---
+title: 'Quick Start (RHEL 8)'
+description: "A walkthrough: setup a demo cluster using Patroni and etcd"
+---
+
+### Description
+
+This section walks you through the various steps to create an highly available **demo** cluster (using **PostgreSQL** version 14 on Rocky Linux 8) where the etcd hosts will be called **etcd1**, **etcd2** and **etcd3**, and the 2 PostgreSQL nodes in _Streaming Replication_: **pg-patroni1** and **pg-patroni2**.
+
+![Demo diagram](images/etcd-demo.png)
+
+---
+
+### Etcd
+
+See the [Etcd installation guide](98-etcd/) to install and setup etcd.
+
+### PostgreSQL
+
+On both **pg-patroni1** and **pg-patroni2** hosts, install PostgreSQL:
+
+```bash
+$ sudo dnf install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-8-x86_64/pgdg-redhat-repo-latest.noarch.rpm
+$ sudo dnf -qy module disable postgresql
+$ sudo dnf install -y postgresql14-server postgresql14-contrib
+$ sudo systemctl disable postgresql-14
+```
+
+Patroni will bootstrap (create) the initial PostgreSQL cluster and be in charge of starting the service.
+
+Don't forget to configure the firewall if needed:
+
+```bash
+$ sudo firewall-cmd --quiet --permanent --add-service=postgresql
+$ sudo firewall-cmd --quiet --reload
+```
+
+### Watchdog
+
+Patroni will be the component interacting with the watchdog device. Set the permissions of the software watchdog on both **pg-patroni1** and **pg-patroni2** hosts:
+
+```bash
+$ cat <<EOF | sudo tee /etc/udev/rules.d/99-watchdog.rules
+KERNEL=="watchdog", OWNER="postgres", GROUP="postgres"
+EOF
+$ sudo sh -c 'echo "softdog" >> /etc/modules-load.d/softdog.conf'
+$ sudo modprobe softdog
+$ sudo chown postgres: /dev/watchdog
+```
+
+### Patroni
+
+On both **pg-patroni1** and **pg-patroni2** hosts, install Patroni and its dependencies for etcd, following the [installation guide](02-installation/).
+
+Let us now define the Patroni configuration in `/etc/patroni.yml`:
```

Comment:
We need to make it clear that it's the same `patroni.yml` on both patroni nodes

Reply:
adding _Except for `$MY_IP` and `$MY_NAME` which are related to the local host, the `patroni.yml` configuration should be the same on all Patroni nodes._

Reply:
fixed in 5bf5f28